components/list/list.html

```diff
@@ -0,0 +1,180 @@
+                            style="width: 1024px; height: 548px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </code>
+                </div>
+                <div class="sub-heading">
+                    <strong>Nested List</strong>
+                </div>
+                <div class="sub-head-content">
+
+                </div>
+                <div class="practical">
+                    <ul class="stacked-list">
+                        <li>
+                            <div class="card-title">Fiction Book</div>
+                            <div class="card-content">"Fiction" refers to literature created from the imagination.
+                            </div>
+                        </li>
+                        <li>
+                            <div class="card-title">Non Fiction</div>
+                            <div class="card-content">Nonfiction is any document or media content that intends, in good
+                                faith, to present only truth and accuracy regarding information, events, or people.
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+                <div class="code-box">
```

Comment:
A suggestion here is that there's an unnecessary scroll. You could adjust the height of the container accordingly which avoids scrolling.
![Screenshot 2022-02-01 at 16 20 34](https://user-images.githubusercontent.com/62737267/151956001-4cc3fc93-670d-45cc-9114-664fd8211de1.png)

Reply:
Let's say if I fix the height. then for small code that will be a big container.

Reply:
When you embed the code, in the iframe you have to change the height for each one individually.
This is the height you should change
 
![Screenshot 2022-02-02 at 09 23 34](https://user-images.githubusercontent.com/62737267/152090628-f93c9afc-e53f-4bcd-94c6-a39bac1f711e.png)